src/commands/convert.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { promises as fPromises } from 'fs';
+import { Command, Flags } from '@oclif/core';
```

Comment:
```suggestion
import { Flags } from '@oclif/core';
import Command from '../base';
```

Use the custom base class we have created for error handling.